web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -90,77 +109,79 @@ const BookingModalForm = props => {
   return (
     <Fragment>
       {composeErrorMessage()}
-      <Form formData={formData} formStatus={formStatus} actions={createCtas()}>
-        <Input
-          type="select"
-          htmlAttrs={{
-            name: 'eventTypeId',
-            options: renderWFH(),
-          }}
-          
-          value={formData.eventTypeId}
-          label="Reason:"
-        />
-        <Input
-          type="date"
-          htmlAttrs={{
-            type: 'input',
-            name: 'start',
-            placeholder: 'Enter a start date',
-          }}
-          value={formData.start}
-          rules={{
-            dateNotInPast: true,
-          }}
-          label={formData.isHalfday ? 'Date' : 'Start Date:'}
-        />
-        <Input
-          type="date"
-          htmlAttrs={{
-            type: 'input',
-            name: 'end',
-            placeholder: 'Enter an end date',
-            disabled: formData.isHalfday,
-          }}
-          value={formData.end}
-          rules={{
-            dateNotInPast: true,
-          }}
-          label="End Date:"
-        />
-        <Input
-          type="input"
-          htmlAttrs={{
-            type: 'input',
-            name: 'employeeRejectionMessage',
-            disabled: !booking.messages,
-          }}
-          value={formData.employeeRejectionMessage}
-          label="Rejection Response:"
-        />
-        <Input
-          type="checkbox"
-          htmlAttrs={{
-            type: 'checkbox',
-            name: 'isHalfday',
-          }}
-          value={formData.isHalfday}
-          label="Request a halfday"
-        />
-        <Input
-          type="input"
-          className={isEventBeingUpdated ? null : 'hide'}
-          htmlAttrs={{
-            type: 'input',
-            name: 'updateMessage',
-            placeholder: 'optional',
-          }}
-          value={formData.updateMessage}
-          label="Reason for updating holiday:"
-          labelClass={isEventBeingUpdated ? null : 'hide'}
-          disabled={!formIsValid}
-        />
-      </Form>
+      {showRemainingHolsAlertMessage() ? holidayAlertMessage() :
+        <Form formData={formData} formStatus={formStatus} actions={createCtas()}>
```

Comment:
Pull the Form out into its own function.  A lot being rendered here off the ternary.